docs/team/kishendranvendarkon.md

```diff
@@ -3,6 +3,36 @@ layout: page
 title: Kishendran Vendar Kon's Project Portfolio Page
 ---
 
-### Project: Spam
+### Project: SPAM
 
-_Coming Soon_
+SPAM is a desktop application that is made for fast-typing marketers to handle large quantities of marketing leads and
+contacts efficiently and precisely. It is a CLI application that has UNIX-like command line syntax, and sports an
+elegant and modern GUI created with JavaFX. The application is written in Java, and has about 15 kLoC.
```

Comment:
Just wondering if it should be `15k LoC` or `15 kLoC`?
@huizhuansam you wrote it the same way

Reply:
Should be the latter. We are following `johndoe.md`.